Go/README.md

```diff
@@ -49,23 +51,31 @@ func main() {
 
 Run the program by running:
 
-    $ go run hello.go
-    Hello World!
+```sh
+$ go run hello.go
```

Comment:
nit: remove the `$`

Reply:
I did a run-through in https://github.com/sb2nov/mac-setup/pull/276. In this case I think it makes sense to keep it as is, because there's output below.

Reply:
Let's use `console` instead of `sh` here.

`console` is good for when there's a prompt and output, `sh` for commands you might find in a script. For example:

```console
$ # with console
$ echo hello
hello
```

```sh
# with sh
echo hello
```

Reply:
Nice, didn't know that. Sounds good to me. Is it OK to do it outside this PR? 🙂